### DISCUSSION TITLE
Patreon or way to donate a few bucks?

### DISCUSSION THREAD

**Discussion by FurrFett** [X3n0m0rph59/eruption#139] [Q&A] [answered]
I found this when looking to get my Elo Air 7.1 to work, and other than a quirky bug where once i installed it it causes the volume slider in the system settings dropdown to be wonky and sometimes i have to turn my device on or off multiple times to get it to connect, it works flawless otherwise.

I havent attempted to use the surround settings (if they are even working yet) because i never did in windows since i dont want the battery drain.  But its pretty impressive so far and i've tried to find a patreon or donation link so i can send a few bucks the dev's way.  does anyone have a link or something where i can do just that?

also, if theres any log files or anything that get generated in the background id be happy to upload them if possible if that would also help development.

**Comment by X3n0m0rph59** [comment, marked as answer]
Hi @FurrFett, thank you for your request!

The Eruption daemon currently only handles battery level reporting and RGB LED lighting on the `ROCCAT Elo 7.1 Air` headset device. Eruption is not directly involved in playing back audio. It is planned to support most of the extended settings of the device sometime in the future. Audio playback and surround sound processing etc. is handled by the Linux kernel and by the PulseAudio / PipeWire daemons.

There is no way to donate to the Eruption project at this time. If I find the time, I will try to set up a "Donate" button or something like that.

To increase the chance of a successful connection of the `ROCCAT Elo 7.1 Air` to the USB dongle, you may want to consider the following:
* The dongle is very picky about USB bandwidth allocation. So don't plug the dongle into a USB hub but use a dedicated USB port on your system for it. This is especially true if you use a wireless mouse with 1000hz poll-rate, or something like that on the same USB port / hub.
* Don't place the dongle too close to the headset
* The dongle sometimes crashes hard, re-plugging or re-booting the pc will fix the connection issue then

Thank you for offering your assistance. It is much appreciated!
